src/main/resources/config/application.yaml

```diff
@@ -33,4 +33,3 @@ powsybl-ws:
 # default values for loadflow providers "Hades2" or "OpenLoadFlow"
 sensitivity-analysis:
   default-provider: OpenLoadFlow
-  results-threshold: 0.01
```

Comment:
The same removal must also be made in deployment : k8s/resources/study/config/sensitivity-analysis-server-application.yml file